app/controllers/users_controller.rb

```diff
@@ -0,0 +1,5 @@
+class UsersController < ActionController::Base
```

Comment:
- [ ] This controller should inherit from the `ApplicationController`, not `ActionController::Base`. Kindly fix this.